[
    {
        "cve": "CVE-2024-42992",
        "created_datetime": "2024-09-03T02:01:15",
        "author_id": "22899183",
        "data_id": "1830788114199257229",
        "reposts": 1,
        "post": "rt @iototsecnews: python library pandas \u306e\u8106\u5f31\u6027 cve-2024-42992\uff1apoc \u306e\u767b\u5834\u3068 nvd \u306e\u00a0reject\nhttps://t.co/dxafusxjgr\n#cyberattack #exploit #github #po\u2026",
        "audience": 3541,
        "audience_total": 3541,
        "total_reposts": 1,
        "post_count": 0,
        "author": "agile cat",
        "username": "agile_cat",
        "v_score": 0.0187496,
        "e_score": 0.00043,
        "base_score": "7.5",
        "description": "REJECTED REASON: DO NOT USE THIS CANDIDATE NUMBER. CONSULTIDS: NONE. REASON: THIS CANDIDATE WAS WITHDRAWN BY ITS CNA. FURTHER INVESTIGATION SHOWED THAT IT WAS NOT A SECURITY ISSUE. NOTES: NONE.",
        "last_modified_datetime": "2024-08-26 15:15:00",
        "published_datetime": "2024-08-23 18:15:00",
        "vendor_logo": null,
        "profile_url": null,
        "feed_name": "twitter"
    }
]